src/index.ts

```diff
@@ -138,20 +160,15 @@ export default class Kumulos {
     }
 
     async pushRegister(): Promise<void> {
-        const mgr = await getPushOpsManager(this.context);
-
-        return mgr
-            .requestNotificationPermission(this.context)
-            .then(perm => {
-                if ('granted' !== perm) {
-                    return Promise.reject(
-                        'Notification permission not granted'
-                    );
-                }
-            })
-            .then(() => {
-                return mgr.pushRegister(this.context);
```

Comment:
I think we've subtly changed the behaviour of this method by not awaiting/returning the result of the pushRegister() call? (masking any errors from it in the rejection chain)

This is a public API of the SDK so this could technically be a breaking change.

Can we achieve the same semantics as before?

Reply:
The promise is not returned from the web SDK so no breaking change